### PR TITLE
Reorganize README with clearer messaging and calls to action

### PR DESCRIPTION
**Pull request by tjvantoll**
Ping @remotesynth, @ikoevska, @jlooper, @valentinstoychev for a few more eyes on this.

Thoughts?

### CODE REVIEW

**Comment by tjvantoll** [comment]
I’m considering putting the whole “Why NativeScript?” part in a wiki page on this repo and linking to it form the main REAME.

File: README.md [hunk 1]
```diff
@@ -1,70 +1,53 @@
...
-- **Twitter** - [@NativeScript](http://twitter.com/NativeScript) 
-- **Blog** - [http://nativescript.org/blog](http://www.nativescript.org/blog)
-- **Forum** - [General discussion](https://groups.google.com/forum/#!forum/nativescript)
-- **StackOverflow** - [http://stackoverflow.com/questions/tagged/nativescript](http://stackoverflow.com/questions/tagged/nativescript)
-- **IRC Chatroom** - [http://webchat.freenode.net/?channels=nativescript](http://webchat.freenode.net/?channels=nativescript)
+# NativeScript [![Build Status](https://travis-ci.org/NativeScript/NativeScript.svg)](https://travis-ci.org/NativeScript/NativeScript)
+
+![NativeScript logo](http://i.imgur.com/YmNIMqS.png)
+
+[NativeScript](http://www.nativescript.org) is a framework for building native iOS and Android apps using JavaScript and CSS. NativeScript renders UIs with the native platform’s rendering engine—no [WebViews](http://developer.telerik.com/featured/what-is-a-webview/)—resulting in native-like performance and UX.
```
Comment (ikoevska) [inline README.md]: The multitude of dashes here is overwhelming :) Please, consider rephrasing this to make it more readable.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,53 @@
...
+- [NativeScript’s home page](http://nativescript.org)
+- [NativeScript’s documentation](http://docs.nativescript.org/)
+- [@NativeScript on Twitter](http://twitter.com/NativeScript) 
+- [NativeScript’s blog](http://nativescript.org/blog)
+- [NativeScript’s community forum](https://groups.google.com/forum/#!forum/nativescript)
+- [NativeScript on Stack Overflow](http://stackoverflow.com/questions/tagged/nativescript)
+
+## Repositories
+
+The NativeScript framework is made up a number of components, all of which are open source and on GitHub. Here are the major ones:
```
Comment (ikoevska) [inline README.md]: made up a -> made up of / consists of

File: README.md [hunk 3]
```diff
@@ -1,70 +1,53 @@
...
+
+- [The current repo](//github.com/NativeScript/NativeScript/) contains the code of the [NativeScript cross-platform modules](http://docs.nativescript.org/core-concepts/modules), which abstract iOS and Android APIs into JavaScript APIs—e.g. `camera.takePicture()`. The modules are written in TypeScript.
```
Comment (ikoevska) [inline README.md]: Read in the npm page of the CLI "the current repo" will look weird. Consider rephrasing it.

File: README.md [hunk 4]
```diff
@@ -1,70 +1,53 @@
...
+- The [iOS Runtime repo](//github.com/NativeScript/ios-runtime/) contains the code that hosts the iOS app, and allows the JavaScript code of your application to be executed on an iOS device. The iOS runtime is written in a fun mix of C++, Objective-C, and more.
```
Comment (ikoevska) [inline README.md]: Just wondering why "The" is enclosed in the square brackets above and not here and below :)

File: README.md [hunk 5]
```diff
@@ -1,70 +1,53 @@
...
+- The [Android Runtime repo](//github.com/NativeScript/android-runtime) contains the code that hosts the Android app, and allows that JavaScript code of your application to be executed on an Android device. The Android runtime is written an even more fun mix of C++ and Java.
```
Comment (ikoevska) [inline README.md]: that JavaScript code of your application -> the JavaScript code.... ?

File: README.md [hunk 6]
```diff
@@ -1,70 +1,53 @@
...
+- The [NativeScript CLI repo](//github.com/NativeScript/nativescript-cli) contains the command-line interface for creating an application using the NativeScript framework. The CLI is written in TypeScript.
+- The [NativeScript Docs repo](//github.com/NativeScript/docs) contain the NativeScript framework documentation which is available on <http://docs.nativescript.org/>. The docs are written in Markdown.
```
Comment (ikoevska) [inline README.md]: contain -> contains

available on -> available at

File: README.md [hunk 7]
```diff
@@ -1,70 +1,53 @@
...
+
+In addition to the code that makes up the NativeScript framework itself, we also provide a number of [open-source sample apps](https://www.nativescript.org/app-samples-with-code) that you can reference while building your NativeScript application.
+
+## Contributing
+
+We love PRs, and accept them for all of our repositories—even docs! Please follow our [contribution guide](https://www.nativescript.org/contribute) if you want to become part of the project.
```
Comment (ikoevska) [inline README.md]: Please, follow

File: README.md [hunk 8]
```diff
@@ -1,70 +1,53 @@
...
+
+## Angular 2
+
+We are [working together with the Google Angular JS team](http://angularjs.blogspot.com/2015/12/building-mobile-apps-with-angular-2-and.html) to make Angular 2.0 work on top of NativeScript. To check our progress please see the following links:
```
Comment (ikoevska) [inline README.md]: To check our progress, please, see

**Comment by ikoevska** [comment]
Hey @tjvantoll, I know this one's merged and closed. However, I've been overwhelmed in the past few weeks and I haven't had the chance to take a look until now. I've left a few comments that you might want to address in the future.

**Comment by tjvantoll** [comment]
@ikoevska: No worries at all. I just implemented your suggested updates. Thanks for getting around to this.

**Pull request closed without merging**